### ISSUE TITLE
Whitespaces in email address check should be ignored

### ISSUE THREAD

**Issue by ankur12-1610** [vector-im/element-web#22756]
### Your use case

The whitespaces in the email address check should be ignored

https://user-images.githubusercontent.com/76884959/177620004-8bc3253a-e3bd-4449-9cfa-b0961d7c8ba7.mp4

**Comment by ankur12-1610** [comment]
This issue is related to PR: https://github.com/matrix-org/matrix-react-sdk/pull/5623

**Comment by turt2live** [comment]
this doesn't need a product decision
